SWLOR.Game.Server/Feature/PerkDefinition/MartialArtsPerkDefinition.cs

```diff
@@ -68,26 +71,40 @@ private void InnerStrength()
                 .Name("Inner Strength")
 
                 .AddPerkLevel()
-                .Description("Improves critical range by 1. [Cross Skill]")
+                .Description("Improves critical chance by +5%. [Cross Skill]")
```

Comment:
For consistency let's remove the plus sign from the description. I don't think we do that anywhere else.